picard/coverart/__init__.py

```diff
@@ -130,7 +137,7 @@ def download_next_in_queue(self):
                     p = provider(self)
                     if p.enabled():
                         log.debug("Trying cover art provider %s ..." % name)
-                        ret = p.queue_downloads()
+                        ret = p.queue_images()
```

Comment:
Do we know if there are any plugins out there that already provide new cover art providers?